r/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN Rscript setup/install_or_die.r BiocManager
 RUN R -e 'BiocManager::install(c("AnnotationDbi", "BiocGenerics", "GO.db", "pcaMethods", "org.Dr.eg.db", "org.Hs.eg.db", "org.Mm.eg.db"))'
 
 RUN R -e 'install.packages("RcppSpdlog")'
-RUN R -e 'devtools::install_github("kharchenkolab/pagoda2")'
+RUN R -e 'devtools::install_github("kharchenkolab/pagoda2@v0.1.4")'
 RUN R -e 'devtools::install_github("kharchenkolab/conos@v1.3.1")'
```

Comment:
is it worth pre-emptively locking the other packages too?

Reply:
I am not sure, @marcellp might have a better idea.

Reply:
My understanding is that it's not possible to do it in the way we are installing stuff just now.

It is possible, although it may be worth having a separate ticket for that.

Reply:
Do you mind writing that ticket up @marcellp? Seems like you have the most context of the work needed and why we need to do it on this one.